### DISCUSSION TITLE
Comparison of GEFS-Aerosols and UFS-COCART in annual simulations

### DISCUSSION THREAD

**Discussion by lipan-NOAA** [ufs-community/ufs-weather-model#3021] [General]
- Simulation time period: September 01, 2019 to August 31, 2020; every day starts from 00z, 24-hour forecast;

- UFS-GOCART initial conditions: 
1. Chemistry: Transformed from GEFS-Aerosols simulation results for day one (September 1, 2019). It's been a warm start ever since.
2. Meteorology: meteorology conditions are updated every 24 hours using the operational GFS output.

- Emissions: GBBEPx and CEDS 2019;

"You can visualize 201910 to 202008 daily forecast between UFS and GEFS against MERRA2 for all species at this site : 
https://www.emc.ncep.noaa.gov/gc_wmb/parthab/UFS_Aerosol/html/fv3_aod_png.html
Also monthly Bias : https://www.emc.ncep.noaa.gov/gc_wmb/parthab/UFS_Aerosol/html/fv3_gridstat_png.html"

**The biggest problem is the overestimation of OC in UFS-GOCART.** 

Any comments and suggestions are welcome

**Comment by lipan-NOAA** [comment]
xiaoyang provided new GB emissions. The data does not have the inconsistency between binary format and netcdf format. The 2017-2020 reprocessed GB emissions for GOCART are archived in "/lfs/h2/emc/physics/noscrub/Li.Pan/data/gocart_emissions/nexus/GBBEPx" on WCOSS2

**Comment by lipan-NOAA** [comment]
Using the new GBBEPx emissions, GEFS-Aero and UFS-Aero were re-run from September 2019 to August 2020, and the comparison of the two models is at the webpage of "https://www.emc.ncep.noaa.gov/gc_wmb/parthab/UFS_Aerosol/html/fv3_gridstatnew_png.html"

**Comment by lipan-NOAA** [comment]
New GBBEPx emissions improve model performance for GEFS-Aero and UFS-Aero;
**UFS-Aero still overestimates OC compared to GEFS-Aero**

**Comment by yangfanglin** [comment]
@lipan-NOAA There are no plots after May 2020 on the webpage.    Does the overestimate of OC becomes worse at longer forecast lead ?    It appears there is also large seasonal variation.

**Comment by lipan-NOAA** [comment]
@yangfanglin  plots after May 2020 will be posted soon. The lifetime of OC is on the order of days, so if biomass emissions are reasonably correct and there are no errors in the model, they will not accumulate incorrectly. The seasonal or monthly variation in aerosols is so large that we must use predictive aerosols as feedback to weather models.

**Comment by NeilBarton-NOAA** [comment]
@lipan-NOAA is the GEFS-Aerosols the GEFSv12 member with Aerosols?

**Comment by lipan-NOAA** [comment]
@NeilBarton-NOAA yes

**Comment by gspetro-NOAA** [comment]
@lipan-NOAA Has this issue been resolved, or is there still work to be done?

**Comment by lipan-NOAA** [comment]
@gspetro-NOAA We are still working on this. About two weeks ago, the new GOCART source code was updated in the UFS and global workflows. We ran sensitivity simulations for the period July 2023 and September 2023.

**Comment by gspetro-NOAA** [comment]
> [@gspetro-NOAA](https://github.com/gspetro-NOAA) We are still working on this. About two weeks ago, the new GOCART source code was updated in the UFS and global workflows. We ran sensitivity simulations for the period July 2023 and September 2023.

Thanks for the update! :)

**Comment by gspetro-NOAA** [comment]
@lipan-NOAA After some discussion with other code managers, we were thinking that this issue is more appropriate as a discussion, so I am going to transfer it to WM Discussions. Whenever a related update to the UFS WM is needed, you can open an issue for that specific purpose. Everything here is valuable discussion, but it's not concretely proposing a bug or enhancement that will become a PR.
